plugins/azure/fix_plugin_azure/resource/base.py

```diff
@@ -473,6 +473,102 @@ class AzureUserAssignedIdentity:
+
+
+@define(eq=False, slots=False)
+class AzureServerBackup:
+    kind: ClassVar[str] = "azure_server_backup"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "backup_interval_hours": S("backupIntervalHours"),
+        "backup_retention_days": S("backupRetentionDays"),
+        "earliest_restore_date": S("earliestRestoreDate"),
+        "geo_redundant_backup": S("geoRedundantBackup"),
+    }
+    backup_interval_hours: Optional[int] = field(default=None, metadata={'description': 'Backup interval hours for the server.'})  # fmt: skip
+    backup_retention_days: Optional[int] = field(default=None, metadata={'description': 'Backup retention days for the server.'})  # fmt: skip
+    earliest_restore_date: Optional[datetime] = field(default=None, metadata={'description': 'Earliest restore point creation time (ISO8601 format)'})  # fmt: skip
+    geo_redundant_backup: Optional[str] = field(default=None, metadata={'description': 'Enum to indicate whether value is Enabled or Disabled '})  # fmt: skip
+
+
+@define(eq=False, slots=False)
+class AzureServerNetwork:
+    kind: ClassVar[str] = "azure_server_network"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "delegated_subnet_resource_id": S("delegatedSubnetResourceId"),
+        "private_dns_zone_resource_id": S("privateDnsZoneResourceId"),
+        "public_network_access": S("publicNetworkAccess"),
+    }
+    delegated_subnet_resource_id: Optional[str] = field(default=None, metadata={'description': 'Delegated subnet resource id used to setup vnet for a server.'})  # fmt: skip
+    private_dns_zone_resource_id: Optional[str] = field(default=None, metadata={'description': 'Private DNS zone resource id.'})  # fmt: skip
+    public_network_access: Optional[str] = field(default=None, metadata={'description': 'Enum to indicate whether value is Enabled or Disabled '})  # fmt: skip
+
+
+@define(eq=False, slots=False)
+class AzureServerHighAvailability:
+    kind: ClassVar[str] = "azure_server_high_availability"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "mode": S("mode"),
+        "standby_availability_zone": S("standbyAvailabilityZone"),
+        "state": S("state"),
+    }
+    mode: Optional[str] = field(default=None, metadata={"description": "High availability mode for a server."})
+    standby_availability_zone: Optional[str] = field(default=None, metadata={'description': 'Availability zone of the standby server.'})  # fmt: skip
+    state: Optional[str] = field(default=None, metadata={"description": "The state of server high availability."})
+
+
+@define(eq=False, slots=False)
+class AzureServerMaintenanceWindow:
+    kind: ClassVar[str] = "azure_server_maintenance_window"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "custom_window": S("customWindow"),
+        "day_of_week": S("dayOfWeek"),
+        "start_hour": S("startHour"),
+        "start_minute": S("startMinute"),
+    }
```

Comment:
base?

Reply:
We use `AzureServerBackup`, `AzureServerDataEncryption`, `AzureServerMaintenanceWindow`, `AzureServerHighAvailability` and `AzureServerNetwork` classes in `mysql` and `postgresql` resources.

Reply:
I would try to keep the base as minimal as possible. Importing classes from MySQL is most probably fine in PostgreSQL (or vice versa) .